website/source/docs/platform/k8s/run.html.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: "docs"
-page_title: "Running Consul - Kubernetes"
+page_title: "Installing Consul on Kubernetes - Kubernetes"
```

Comment:
I felt Installing was better than Running since that's what I'd click on looking for installation instructions. We use Installing Consul in the top level docs as well.